### PR TITLE
Configure tests etc

### PR DESCRIPTION
**Pull request by weistrand**
### WHY are these changes introduced?

Tests not runnable in project yet. 

### WHAT is this pull request doing?

Make tests runnable by adding test config etc. 

### HOW can this pull request be tested?

Specs? Manual steps? Please fill me in.

---

Friendly reminders

- Include reference to Trello (or possibly Slack)
- Include changelog to support if applicable
- Lint rules pass
- The environment (`heroku config`) has been updated if needed (new `ENV` variables)

### CODE REVIEW

**Comment by snichme** [comment]
Nice, good to be able to run the tests :) 
Just add the coverage dir to .gitignore

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+source "https://rubygems.org"
+
+ruby File.read(".ruby-version").strip
+
+gem "dotenv"
```
Comment (dentarg) [inline Gemfile:5]: Don't think `dotenv` is needed?

**Comment by dentarg** [comment]
@weistrand This is a library so we do not need to commit `Gemfile.lock`, instead add it to .gitignore

**Comment by dentarg** [comment]
I can do it :)

**Comment by dentarg** [comment]
Done with https://github.com/84codes/prometheus_parser/pull/5

**Pull request merged**